config/config.example.yml

```
@@ -823,6 +823,15 @@ default_user_preferences:
   ##
   #save_player_pos: false
 
+  ##
+  ## Show the channel's watermark on the bottom-right corner of videos
+  ##
+  ## Accepted values: true, false
+  ## Default: false
+  ##
+  #show_channel_watermark: true
```

```suggestion
  #show_channel_watermark: false
```

This too, unless you wanted the default to be true?

---

Yeah those are intentional. I'd like for it to be enabled by default with an option to disable it for those who don't want it.

But if enough people want it be disabled by default, I'll change that.

---

@syeopite ah okay, i wasnt sure if it was intentional or not as it says `Default: false` on line 830 in thia file (which would need to change in that case)

---

Oops missed that. Thanks!